geode-core/src/main/java/org/apache/geode/internal/JarDeployer.java

```diff
@@ -50,17 +48,21 @@
 public class JarDeployer implements Serializable {
   private static final long serialVersionUID = 1L;
   private static final Logger logger = LogService.getLogger();
-  public static final String JAR_PREFIX_FOR_REGEX = "";
+  // The pound version scheme predates the sequenced version scheme
+  private static final Pattern POUND_VERSION_SCHEME =
+      Pattern.compile("^vf\\.gf#(?<artifact>.*)\\.jar#(?<version>\\d+)$");
+  // The sequenced version scheme predates the semantic version scheme. We use this scheme when
```

Comment:
These javadocs are now wrong. We always use this scheme even if it has a semantic version